### ISSUE TITLE
Why can't my innerText get the text？

### ISSUE THREAD

**Issue by andrew2558** [hlaueriksson/puppeteer-sharp-contrib#4]
Why can't my innerText get the text, run directly, the program is stuck, quick check, prompt: item.InnerText () The function evaluation requires all threads to run. String

```csharp
await page.EvaluateExpressionAsync("document.querySelector(\"input[name='q']\").value=''");
await page.TypeAsync("input[name='q']", w, new PuppeteerSharp.Input.TypeOptions { Delay = 5 });
await page.Keyboard.PressAsync("Enter");
await page.WaitForNavigationAsync();
await page.WaitForTimeoutAsync(1500);
var list = await page.QuerySelectorAllAsync("#webPhrase>p");

foreach (var item in list)
{
     Console.WriteLine(item.InnerText());
}
```

**Comment by hlaueriksson** [comment]
Thanks for your feedback!

Try the async version:
`await item.InnerTextAsync()`

**Comment by andrew2558** [comment]
thanks hlaueriksson

**Comment by hlaueriksson** [comment]
I just released version `2.0.0` of [PuppeteerSharp.Contrib.Extensions](https://www.nuget.org/packages/PuppeteerSharp.Contrib.Extensions/).
Stick to this package to avoid the risk of deadlocks.

All the unsafe _sync over async_ extension methods has been moved to the `PuppeteerSharp.Contrib.Extensions.Unsafe` package.
